scenariolib/visit.go

```diff
@@ -170,7 +175,8 @@ func (v *Visit) sendSearchEvent(q, actionCause, actionType string, customData ma
 		}
 	}
 
-	if v.Config.AllowEntitlements { // Custom shit for besttech, I don't like it
```

Comment:
I changed formating in Intellij so that I now use go fmt, this will never happen again.